File: application/src/java/wandledi/config/Bootstrap.java
package wandledi.config;

import javax.servlet.ServletContext;
import wandledi.example.controllers.HomeJsp;
import wandledi.example.controllers.Home;
import wandledi.java.WandlediBootstrap;

/**
 *
 * @author Markus Kahl
 */
public class Bootstrap extends WandlediBootstrap {

    public void init(ServletContext servletContext) {

        setPersistenceUnit("MyPU");
        addControllers(HomeJsp.class, Home.class);
    }

    public void destroy() {
    }
}

File: scala-app/src/main/scala/wandledi/config/Bootstrap.scala
package wandledi.config

import wandledi.java.WandlediBootstrap
import javax.servlet.ServletContext

import wandledi.scala.example.controllers._

class Bootstrap extends WandlediBootstrap {

  def init(servletContext: ServletContext) {

    setPersistenceUnit("ScalaPU")
    addControllers(classOf[Home])
  }

  def destroy() {

  }
}
